chart/elastalert2/README.md

```diff
@@ -82,6 +82,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `serviceAccount.annotations`                 | ServiceAccount annotations                                                                                                    |                                                          |
 | `podSecurityPolicy.create`                   | [DEPRECATED] Create pod security policy resources                                                                             | `false`                                                  |
 | `resources`                                  | Container resource requests and limits                                                                                        | {}                                                       |
+| `rulesVolumeName`                            | Specifies the rules volume to be used                                                                                         | "rules"                                                  |
```

Comment:
This description could be improved to help readers see why this could be beneficial. For example you could add something like this text: `Useful for mounting a custom rules folder via the extraVolumes parameter, instead of using the default rules configMap or secret rule mounting method.`.

Reply:
yes, I will add it

Reply:
added it